src/test/java/com/guysfromusa/carsgame/control/CollisionMonitorTest.java

```diff
@@ -0,0 +1,90 @@
+package com.guysfromusa.carsgame.control;
+
+import com.guysfromusa.carsgame.entities.enums.CarType;
+import com.guysfromusa.carsgame.v1.model.Car;
+import com.guysfromusa.carsgame.v1.model.Point;
+import org.junit.Test;
+
+import java.util.List;
+
+import static com.guysfromusa.carsgame.entities.enums.CarType.MONSTER;
+import static com.guysfromusa.carsgame.entities.enums.CarType.RACER;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by Konrad Rys, 03.05.2018
+ */
+public class CollisionMonitorTest {
+
+    private CollisionMonitor collisionMonitor = new CollisionMonitor();
+
+    @Test
+    public void shouldMarkCarsCrashedWhenTypeRacing(){
+        //given
+        Car car1 = createCar(1, 1, "car1", RACER);
+        Car car2 = createCar(1, 1, "car2", RACER);
+        List<Car> cars = asList(car1, car2);
+
+        //when
+        collisionMonitor.execute(cars);
+
+        //then
+        assertThat(cars).extracting(Car::isCrashed).containsOnly(true, true);
```

Comment:
please wrap the line to keep consistent formatting: 

assertThat(cars)
.extracting(Car::isCrashed)
.containsExactly(true, true);